Content.Client/Construction/ConstructionMenuPresenter.cs

```diff
@@ -250,8 +251,8 @@ private void GenerateStepList(ConstructionPrototype prototype, ItemList stepList
                             stepList.AddItem(
                                 !firstNode
                                     ? Loc.GetString(
-                                        "{0}. Add {1}x {2}.", stepNumber++, materialStep.Amount, materialStep.Material)
-                                    : Loc.GetString("      {0}x {1}", materialStep.Amount, materialStep.Material), icon);
+                                        "{0}. Add {1}x {2}.", stepNumber++, materialStep.Amount, materialStep.MaterialPrototype.ID)
+                                    : Loc.GetString("      {0}x {1}", materialStep.Amount, materialStep.MaterialPrototype.ID), icon);
```

Comment:
Rather than using the Material Prototype ID, you should add a human-readable name property to the prototype

Reply:
Done in https://github.com/space-wizards/space-station-14/pull/3387/commits/47f31aee69126d9ca268f2e7872f494e9bbbf3cd